src/main/webapp/WEB-INF/user.jsp

```diff
@@ -38,11 +48,22 @@
       <br/>
       <textarea name="about-me" placeholder="About me" id="about-me-input"></textarea>
       <br/>
-      <input type="submit" value="Submit">
+      <input type="submit" value="Submit" >
     </form>
     <hr/>
     <% }  %>
-
+    <% if(isUserLoggedIn){ %>
+    <form id="message-form" action="<%= uploadUrl %>" method="POST" enctype="multipart/form-data">
+    Enter a new message:
+    <br/>
+    <textarea name="text" placeholder="Enter a message" id="message-input"></textarea>
+    <br/>
+    Add an image to your message:
+    <input type="file" name="image">
+    <br/>
+    <input type="submit" value="Submit">
+    </form>
+    <% }  %>
```

Comment:
Another merge issue - this form should no longer display on the user page, as we can't post messages to users any more